db/migrate/20170116210702_add_manager_ref_to_authentications.rb

```diff
@@ -0,0 +1,5 @@
+class AddManagerRefToAuthentications < ActiveRecord::Migration[5.0]
+  def change
+    add_column :authentications, :manager_ref, :string
```

Comment:
I dont think any  other manager than the new ansible_tower ever refreshed `authentications`

```bash
❯ git grep -i auth -- app/models/ems_refresh
app/models/ems_refresh/save_inventory.rb:          link_habtm(found, key_backup[:key_pairs], :key_pairs, ManageIQ::Providers::CloudManager::AuthKeyPair)
app/models/ems_refresh/vc_updates.rb:      "config.storageDevice.hostBusAdapter[*].authenticationProperties.chapAuthEnabled",
```

So, no problem, I guess...